contracts/exchange/tslint.json

```diff
@@ -2,5 +2,8 @@
     "extends": ["@0x/tslint-config"],
     "rules": {
         "custom-no-magic-numbers": false
+    },
+    "linterOptions": {
+        "exclude": ["src/artifacts.ts"]
```

Comment:
Why are we excluding this? Can we just use a `tslint:disable:next-line` for the specific line we're trying to address?

Reply:
It's a generated file. For some reason it generates the imports out of order, which violates linter rules.